<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepoMan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QSplitter" name="centerSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="modeLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="changesModeBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	border:none;
	outline: none;
	text-align: left;
	padding-left: 14px;
	font-size: 14px;
}
QPushButton:checked{
	background:LightGray;
}</string>
            </property>
            <property name="text">
             <string>Changes</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/action_changes.svg</normaloff>:/resources/action_changes.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="historyModeBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	border:none;
	outline: none;
	text-align: left;
	padding-left: 14px;
	font-size: 14px;
}
QPushButton:checked{
	background:LightGray;
}</string>
            </property>
            <property name="text">
             <string>History</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/action_history.svg</normaloff>:/resources/action_history.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RefTreeView" name="refTreeView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeView {
		background-color: #EFEFEF;
}
QTreeView::item { 
	height: 28px;
}
QTreeView::item:focus {
	border:none;
}
QTreeView::item:selected {
	background: LightGray;
}
QTreeView::branch:selected {
    background: LightGray;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="indentation">
           <number>10</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QStackedWidget" name="rightPanel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_Open"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Configurations"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuRepo">
    <property name="title">
     <string>Repo</string>
    </property>
    <addaction name="actionRepo_Switch_manifest"/>
    <addaction name="actionRepo_Sync"/>
    <addaction name="actionRepo_Start"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRepo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFile_Open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionFile_Configurations">
   <property name="text">
    <string>Configuration File</string>
   </property>
  </action>
  <action name="actionFile_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRepo_Switch_manifest">
   <property name="text">
    <string>Switch manifest</string>
   </property>
  </action>
  <action name="actionRepo_Sync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionRepo_Start">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionHelp_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RefTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/reftreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
